amf-client/shared/src/test/resources/org/raml/parser/types/builtin/number-float/output.txt

```diff
@@ -4,6 +4,6 @@ types:
   MyCustomType:
     type: number
     minimum: 2.5
-    maximum: 9.3
+    maximum: 9.30
```

Comment:
Why it adds a 0 now? As I could see, the method to avoid this in the BaseEmitters is still there

Reply:
9.30 is actually the original value, previously it was removed when calling .toDouble. It is now being maintained because BigDecimal.toString conserves zeros to the right. This can be adjusted easily with .stripTrailingZeros(), but i don't think there is a need in removing them right?